security/registries.adoc

```diff
@@ -25,48 +24,39 @@ of protection by using trusted sources.
 
```

Comment:
With the "as discussed" para above, I'd suggest it be more like "As discussed in the Container Content section...". As is it, I feel like it is relying on the reader reading this like a book and having that section directly above them, which is something we're trying to get away from.